src/features/current_event/atoms/currentEventResource.ts

```diff
@@ -29,7 +30,13 @@ export const currentEventResourceAtom = createAsyncAtom(
         `/events/${deps.feed.id}/${deps.event.id}`,
         undefined,
         undefined,
-        { signal: abortController.signal, errorsConfig: { dontShowErrors: true } },
+        {
+          signal: abortController.signal,
+          errorsConfig: {
+            dontShowErrors: false,
```

Comment:
I expect now you will always see error when request canceled, that I guess not what you want